### PR TITLE
fix install and update to latest version

### PR DESCRIPTION
**Pull request by vasu1124**

### CODE REVIEW

**Review by In-Ko — APPROVED** [review]
/LGTM

File: install.sh [hunk 1]
```diff
@@ -1,32 +1,21 @@
 #!/bin/sh -e
 
-WORKSPACE="${GITHUB_WORKSPACE:=/usr/local}"
 WORKSPACE="/usr/local"
-VERSION=${version:=v0.1.0-alpha.2}
```
Comment (mandelsoft) [inline install.sh:5]: This is for using the script outside of a gh action and should be kept with the appropriate version

File: install.sh [hunk 2]
```diff
@@ -1,32 +1,21 @@
...
-if [ "$REPO" != "gardener/component-cli" ]; then
-  ARCHIVESUFFIX=.tgz
-  ARCHIVEFILE="ocm-$PLATFORM$ARCHIVESUFFIX"
-else
-  ARCHIVESUFFIX=.gz
-  ARCHIVEFILE="componentcli-$PLATFORM$ARCHIVESUFFIX"
-fi
-FILE="$(basename "$ARCHIVEFILE" $ARCHIVESUFFIX)"
+VERSION=${version:=v0.2.0-rc.1}
+PLATFORM=Linux_x86_64
```
Comment (mandelsoft) [inline install.sh]: No, we should definately use the official GOOS and GOARCH names, here

**Review by mandelsoft — CHANGES_REQUESTED** [review]

File: install.sh [hunk 3]
```diff
@@ -1,32 +1,21 @@
...
+ARCHIVESUFFIX=.tar.gz
+ARCHIVEFILE="ocm_$PLATFORM$ARCHIVESUFFIX"
```
Comment (nikimanoledaki) [inline install.sh]: The formatting for the archive file's name changed to `"{{ .Binary }}-{{ .Version }}-{{ .Os }}-{{ .Arch }}"` as per @mandelsoft's feedback here: https://github.com/open-component-model/ocm/pull/291#discussion_r1132213989

File: install.sh [hunk 4]
```diff
@@ -1,32 +1,21 @@
...
-PLATFORM=linux-amd64
-if [ "$REPO" != "gardener/component-cli" ]; then
-  ARCHIVESUFFIX=.tgz
-  ARCHIVEFILE="ocm-$PLATFORM$ARCHIVESUFFIX"
-else
-  ARCHIVESUFFIX=.gz
-  ARCHIVEFILE="componentcli-$PLATFORM$ARCHIVESUFFIX"
-fi
-FILE="$(basename "$ARCHIVEFILE" $ARCHIVESUFFIX)"
+VERSION=${version:=v0.2.0-rc.1}
```
Comment (nikimanoledaki) [inline install.sh]: @morri-son and I spoke earlier about cutting a release for `v0.2.0` today/soon so this RC version would become outdated soon... There is no harm in leaving this for now but we probably want to add a way to update the version automatically when there is a new OCM CLI release, right?

**Pull request merged**